extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -115,28 +115,42 @@ function render_player( $player_data, $attributes ) {
 		$player_data
 	);
 
+	$secondary_colors  = get_colors( 'secondary', $attributes, 'color' );
+	$background_colors = get_colors( 'background', $attributes, 'background-color' );
```

Comment:
Let's pass `secondaryColor` and `backgroundColor` as the first parameter to make it just a little less closely coupled.

Reply:
If we change that then this implementation will get more complex. The function uses the first argument to define `has-{$name}`:

```php
$color = get_colors( 'secondary', $attributes, 'border-color' )'

array(
    'class' => 'has-seconday' // <- this class is generated based on the color name
    'style' => 'border-color: #65ac45'
);
```

Reply:
Ah, I didn't catch that. Sounds good 👍